docs/normen/privacy en gegevensbescherming/Beginsel van eerlijkheid (Fairness).md

```diff
@@ -0,0 +1,41 @@
+---
+title: Beginsel van eerlijkheid (Fairness)
+
+---
+
+!!! info "Disclaimer"
+
+    Het Algoritmekader is nog volop in ontwikkeling. Op deze plek willen we vooral aan de slag gaan op een open en transparante wijze. Het is dus niet definitief. Dat betekent dat er dingen opstaan die niet af zijn en soms zelfs fout. Mocht er iets niet kloppen, laat het ons weten via [GitHub](https://github.com/MinBZK/Algoritmekader).
+
+
+## Norm
+**De verwerking van persoonsgegevens op een eerlijke en rechtmatige manier moet gebeuren.**
+
+## Risico
+
+
+## Bronnen
+
+#### Wet- en regelgeving
+
+- [Algemene Verordening Gegevensbescherming (AVG)](https://eur-lex.europa.eu/legal-content/NL/TXT/HTML/?uri=CELEX:32016R0679&qid=1685451198313), artikel 5 en 6, recital 39 en 60 AVG
+
+#### Toetsingskaders
+
+
+## Toelichting
+Individuen moeten op een eerlijke manier worden behandeld bij de verwerking van hun persoonsgegevens. Dit vereist ook transparantie, wat betekent dat betrokkenen op de hoogte moeten worden gesteld van hoe hun gegevens worden verzameld, gebruikt en verwerkt.
```

Comment:
Meer in het algemeen: deze zinnen zijn lijdend 'moeten worden gehandeld' en 'betrokkenen op de hoogte moeten worden gesteld'. Is het een idee dat actiever te maken? 
Bijvoorbeeld: 

De overheid moet individuen op een eerlijke manier behandelen bij de verwerking van hun persoonsgegevens. Dit vereist ook transparantie: dat betekent dat de organisatie betrokkenen op de hoogte moet stellen van hoe hun gegevens worden verzameld, gebruikt en verwerkt.